api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1717,9 +1717,13 @@ type RequiredAddressSet struct {
 // CrashLoopContainerObject specifies crash-loop target for specific container.
 type CrashLoopContainerObject struct {
 	// Name of the target container.
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=253
 	ContainerName string `json:"containerName,omitempty"`
 
 	// Target processes to kill inside the container.
+	// +kubebuilder:validation:MinLength=0
+	// +kubebuilder:validation:MaxLength=10000
 	Targets []ProcessGroupID `json:"targets,omitempty"`
```

Comment:
Those are the wrong annotations:

```text
+kubebuilder:validation:MaxLength
int
specifies the maximum length for this string.
```

Since this is a slice you have to use `+kubebuilder:validation:MaxItems` and `+kubebuilder:validation:MinItems`